recipe/recipe.yaml

```diff
@@ -0,0 +1,87 @@
+context:
+  name: py-rattler
+  python_name: py_rattler
+  version: "0.9.0"
+
+recipe:
+  name: py-rattler-split
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ python_name }}-${{ version }}.tar.gz
+  sha256: 488922a45ed3276732f9c413617b6ed6587eb0583af28e9c5ab48e946e8b1a25
+
+build:
+  number: 1
+
+outputs:
+  - package:
+      name: py-rattler
+    build:
+      # only build for the minimal python version
+      skip: not match(python, "3.9.*")
+      script: build_base
+      python:
+        version_independent: true
+    requirements:
+      build:
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - maturin >=1.2.2,<2
+        - ${{ compiler('c') }}
+        - ${{ stdlib("c") }}
+        - ${{ compiler('rust') }}
+        - cargo-bundle-licenses
+      host:
+        - python
+        - python-abi3
+        - maturin >=1.2.2,<2
+        - pip
+        - if: unix
+          then:
+            - openssl
+      run:
+        - python
+    tests:
+      - python:
+          imports:
+            - rattler
+          python_version: "3.9"
+
+  - package:
+      name: py_rattler
+    build:
+      noarch: generic
+      skip: not (linux and x86_64 and match(python, "3.9.*"))
```

Comment:
Is the python match needed? Since `python` is not used in the package, wouldn't it be build once anyway?